builder/Dockerfile

```diff
@@ -53,6 +53,18 @@ COPY --from=envoy /usr/local/bin/envoy /usr/local/bin/envoy
 
 ENV KUBECONFIG=/buildroot/kubeconfig.yaml
 
+# The YAML parser is... special. First, we need to install libyaml and Cython. No,
+# I don't know why it's called yaml-dev.
+RUN apk --no-cache add yaml-dev cython
+
+# After that, we can download !*@&#!*@&# PyYAML...
+RUN curl -O -L http://pyyaml.org/download/pyyaml/PyYAML-5.1.tar.gz
+RUN tar xzf PyYAML-5.1.tar.gz
```

Comment:
I think the reason that CI is unhappy is that PyYAML is getting extracted over our sources in `/buildroot`.  Perhaps do this in `/tmp/pyyaml`?

Reply:
Oh, nice catch! Done, and I folded the download/build/install `RUN`s together for you too...